_posts/core/2024-12-02-input-validation.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: "Input Validation"
+tags: prime1 prime2
+---
+When an error happens when exporting the game, it's important to make sure the game files you are using are the originals with no modifications. Randovania only supports unmodified game files.
+
+# Verifying your game files
+
+## Metroid Prime and Metroid Prime 2: Echoes
+
+Both games only support the GameCube versions of the game, with uncompressed files in the ISO format.
```

Comment:
Mention that you can use dolphin to verify your input ISO (right click the game -> properties -> verify tab -> verify integrity button)

Also the user should have acccess to the clean hashes so they know they have a good dump (can probably copy paste them from the rdv repo)

Reply:
I don't want to have the hashes duplicated here, that seems terrible

Reply:
could maybe get the hashes dynamically here somehow
But the end user should be able to see the clean supported hashes here for UX purposes.